dockstore-common/src/main/java/io/dockstore/common/Constants.java

```diff
@@ -24,8 +24,8 @@
  */
 public class Constants {
     public static final String WEBSERVICE_BASE_PATH = "webservice.base_path";
-    public static final String WEBSERVICE_TOKEN = "webservice.token";
-    public static final String WEBSERVICE_TOKEN2 = "webservice.token2";
+    public static final String WEBSERVICE_TOKEN_USER_1 = "webservice.tokenUser1";
```

Comment:
Please add a comment explaining what these tokens are and what they can be used for